manifests/params.pp

```diff
@@ -29,9 +29,18 @@
   if $::operatingsystem == 'Ubuntu' {
     if versioncmp($::lsbdistrelease, '8.04') < 1 {
       $init_style = 'debian'
-    } else {
+    } elsif versioncmp($::operatingsystemrelease, '15.04') < 0 {
       $init_style = 'upstart'
     }
+    else {
```

Comment:
@asasfu What do you think about actually getting rid of the fact `is_systemd` and making 15.04 or higher default `systemd` like the rest of the operating systems? The reason being `params.pp` lists the "default" for an OS, in this case the default for 15.04 or higher being `systemd`. If a user chooses to use `upstart` then they can override it via `$consul::init_style`.

I would like to hear @solarkennedy thoughts on this as well.

```
if versioncmp($::lsbdistrelease, '8.04') < 1 {
  $init_style = 'debian'
} elsif versioncmp($::lsbdistrelease, '15.04') < 0 {
  $init_style = 'upstart'
} else {
  $init_style = 'systemd'
}
```